### ISSUE TITLE
Critic 1 loss very different than Critic 2 loss

### ISSUE THREAD

**Issue by canteli** [sfujim/TD3#16]
Thank you for the repository. I just implemented TD3 and observed that the loss of Critic 1 is consistently much higher than the loss of Critic 2. Both networks have the same architecture and learning rates in my code, why may this be? Shouldn't they have similar losses and updates?
![image](https://user-images.githubusercontent.com/29544463/69388797-b123ee00-0c8f-11ea-9469-5ef15e3801fa.png)

**Comment by sfujim** [comment]
Sorry for the delayed response. I wanted to check this out myself / confirm these results. Very interesting find!

The difference between Q1 and Q2 is that the actor pi is trained with respect to Q1. This means we should expect the overestimation of Q1(s,pi(s)) to be greater than Q2(s,pi(s)). Therefore we have min(Q1,Q2) = Q2 more often than Q1. Now presumably this means that the error of Q2 will be lower than Q1. 

Interestingly enough, you can reduce the loss of Q1 by having the actor trained with respect to both Q1 and Q2, but I haven't found any meaningful difference in results doing this. 

Hope that answers your question. Thanks for bringing this to my attention!
